components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/dto/DocumentListDTO.java

```diff
@@ -29,7 +28,34 @@
   
   private List<DocumentDTO> list = new ArrayList<DocumentDTO>();
 
-  
+
+  private String lastUpdatedTime = null;
+
+  private String createdTime = null;
+
+  /**
+  * gets and sets the lastUpdatedTime for DocumentListDTO
+  **/
+  @JsonIgnore
+  public String getLastUpdatedTime(){
+    return lastUpdatedTime;
+  }
+  public void setLastUpdatedTime(String lastUpdatedTime){
```

Comment:
Check all DTO classes. There are places with additional new lines and also some methods are missing preceding empty line before method signature.